index.html

```diff
@@ -6,9 +6,16 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Git_test</title>
 </head>
-<body>
+<body class="body">
     <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Blanditiis debitis culpa eum architecto dolorum qui.
+
+       <ul>
+        <li>hi</li>
+        <li>hello</li>
+        <li>hello there!</li>
+        <li>general kenobi</li>
+       </ul>
```

Comment:
Unordered list and List items are inside the Paragraph tag, if this was a mistake please change it. Other than that the code is great.

Reply:
working on it, thank you.